fullstack/README.md

```diff
@@ -0,0 +1,64 @@
+
+The test is intended to be completed at home in no more than 1-2 hours. It is not timed, so if you want to take more time to showcase your coding skills and ability you may do so (note this is not mandatory, nor expected).
+
+If you haven't finished in that time, you can send us your progress - we don't want you to spend days on this. We understand that your time is valuable, and appreciate any time that you contribute towards building a strong team here.
+
+If you **really cannot spare the time**, you may want to take a look at Option 2.
+
+## Option 1
+
+Build a JavaScript application that shows the latest news from the United Kingdom, using the [newsapi.org](https://newsapi.org) service.
+
+Running the app we should be able to see the latest **UK news** and give the ability to the user to filter them by categories. Given the fact that this is a FullStack challenge, we are expecting a Frontend application and a Backend application that communicate with each other.
+
+A FullStack developer is someone with familiarity in each layer; you might be stronger on the frontend, or vice-versa. Feel free to focus more on what you know best, be creative and show us your skills!
+
+### What we are looking for
+
+* We anticipate that the website may have future requirements, so you should aim for maintainability with your solution
+* The JavaScript ecosystem changes quickly, so we like to see knowledge of JavaScript fundamentals
+* We want to discover how you write code day-to-day (from the folder structure, testing, naming point of view)
+* We don't want you to spend hours tweaking every pixel. Keep it simple.
+* Keep it simple (again :) )
+
+### Submission
+
+You can submit your code however you want (github/bitbucket/gitlab repo - git bundle - zip file)- send the details to your point of contact.
+
+Please include a README containing any setup instructions and keep the setup steps simple.
+
+### FAQ
+
+Q: Shall I provide my `newsapi.org` token?  
+A: Absolutely not. Provide a way we can easily plug it in
+
+Q: What browsers must I support?  
+A: Latest Firefox & Chrome. Don’t worry about any browser inconsistencies.
+
+Q: What version of node must I support?  
+A: >= 8
+
+Q: Should I use Docker?  
+A: It's up to you
+
+Q: Should I deploy the application somewhere it is publicly reachable?
+A: It's up to you
+
+Q: Can I use transpilers?  
+A: Yes, but you [may not need to](http://kangax.github.io/compat-table/es6/).
+
+Q: Can I use libraries, frameworks, etc?  
+A: Yes...but please consider the 'What we are looking for?' guidelines.
+
+Q: Should my solution be client-side only/server-side only/client-server?  
+A: That's up to you. We are interested in how you solve the problem, so we don't want to force you down any particular route.
+
+## Option 2
+
+If you have some personal code that you would be prepared to share with us, we can assess that instead. The code should meet the following criteria:
```

Comment:
"existing code" instead of "personal code".

"look at" instead of "assess"

Reply:
"Anything you share with us will not be shared out of CNI or used for any purposes out of this exercise."

Reply:
👍